testsuite/features/build_validation/core/allcli_sanity.feature

```diff
@@ -310,12 +310,12 @@ Feature: Sanity checks
     And "sle12sp5_buildhost" should communicate with the server using public interface
     And the clock from "sle12sp5_buildhost" should be exact
 
-@sle15sp3_buildhost
+@sle15sp4_buildhost
   Scenario: The SLES 15 SP3 build host is healthy
```

Comment:
SP4

Reply:
Thanks Eric. Fixed and amended the commit since it was so small.